Modix.Data/ModixContext.cs

```diff
@@ -16,6 +16,7 @@ namespace Modix.Data
 
     public class ModixContext : DbContext
     {
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
```

Comment:
Does `= null!` not work here, with explicit constructors?

Reply:
It does, but I did this style to minimize the number of `= null!;`, since that was a lot of fields to do that on. I can change to the other style if you prefer.

Reply:
I for one have no problem with it, it's the "official" way to bypass the check according to EF docs. And it sure beats `#pragma`s in my book.

Reply:
Nono, I mean I prefer using `= null!;` personally. I hate `#pragma`s. Also, I think there's a small amount of value in annotating each one specifically as "hey, here's all the ones that you'd better be sure are initialized, before you try and call them." I.E. "hey, make sure and call `.Include()` for these."